ClassTypes/PowerShell/EthicalHacking/Create-EthicalHackingLabplanAndLab.ps1

```diff
@@ -44,20 +57,23 @@ trap {
 #
 
 # Download AzLab module file, import, and then delete the file
+if (-not (Get-Module -ListAvailable -Name 'Az')) {
+    Import-Module Az -Force 
+}
 
-Import-Module Az.LabServices -Force
+$ClassName = "EthicalHacking"
```

Comment:
Should we add validation around ClassName (or add comments in the code), that it must be a good name (no spaces, etc)?